packages/server/src/app.js

```diff
@@ -29,6 +32,9 @@ app.use(cookieParser());
 app.use(express.static(path.join(__dirname, "public")));
 
 app.use("/", indexRouter);
+app.use("/api/posts", indexRouter);
```

Comment:
mayhaps we could implement a router for each resource

Reply:
look now mustache man